timber-lint/src/main/java/timber/lint/WrongTimberUsageDetector.java

```diff
@@ -65,7 +65,10 @@
       checkNestedStringFormat(context, call);
       return;
     }
-    if ("tag".equals(methodName) && evaluator.isMemberInClass(method, "timber.log.Timber")) {
+    // As of API 24, Log tags are no longer limited to 23 chars
```

Comment:
Watch your periods. [Comments should be sentences](https://nedbatchelder.com/blog/201401/comments_should_be_sentences.html).

Reply:
Drat. Again. Such a bad habit.  😞

Reply:
Wait until you send a comma-separated list without an oxford comma. You will feel my full wrath of comment nit picking!